test/k8sT/istio.go

```diff
@@ -36,13 +36,14 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sIstioTest", func() {
 		// - remind how to test with prerelease images in future
 		// - cause CI infra to prepull these images so that they do not
 		//   need to be pulled on demand during the test
-		// " --set values.pilot.image=quay.io/cilium/istio_pilot:1.10.4" + prerelease +
-		// " --set values.global.proxy.image=quay.io/cilium/istio_proxy:1.10.4" + prerelease +
-		// " --set values.global.proxy_init.image=quay.io/cilium/istio_proxy:1.10.4" + prerelease +
+		// " --set values.pilot.image=quay.io/cilium/istio_pilot:1.10.6" + prerelease +
+		// " --set values.global.proxy.image=quay.io/cilium/istio_proxy:1.10.6" + prerelease +
+		// " --set values.global.proxy_init.image=quay.io/cilium/istio_proxy:1.10.6" + prerelease +
 		// " --set values.global.proxy.logLevel=debug" +
 		// " --set values.global.logging.level=debug"
 		// " --set values.global.mtls.auto=false"
 		ciliumOptions = map[string]string{
+			"hostServices.hostNamespaceOnly": "true",
```

Comment:
Why is this now required? How did the test pass before on CI jobs that run with KPR enabled?

Reply:
Not sure is KPR was enabled in the tests or not, TBH. I reverted this change as KPR should *now* be disabled by default.

Reply:
KPR is disabled by default for Cilium installations, but in end-to-end tests, we explicitly enable it for some CI jobs (net-next and 4.19 ones). AFAICT, the test is passing on those CI jobs as well: https://jenkins.cilium.io/job/cilium-master-k8s-1.23-kernel-net-next/460/testReport/Suite-k8s-1/23/.

If you want to enable this option only when KPR is enabled, you can do so with `helpers.RunsWithKubeProxyReplacement()`.